plugins/modules/hostgroup.py

```diff
@@ -55,6 +55,13 @@
   parameters:
     description:
       - Hostgroup specific host parameters
+  ansible_roles:
```

Comment:
Can you please add a "version_added: 2.1.0" to this section and also a changelog snippet? ;)